cves/kernel/CVE-2021-3744.yml

```diff
@@ -326,8 +338,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: Herbert Xu signed off the fix that was reported by minihanshen.
```

Comment:
additionally it was also signed off by an oracle employee and tested-by/reviewed-by an amd employee https://github.com/torvalds/linux/commit/505d9dcb0f7ddf9d075e729523a33d38642ae680